crates/re_ui/src/list_item2/property_content.rs

```diff
@@ -31,6 +30,8 @@ pub struct PropertyContent<'a> {
 }
 
 impl<'a> PropertyContent<'a> {
+    const COLUMN_SPACING: f32 = 12.0;
```

Comment:
docstring!

is this spacing between all columns, including the third button column?

Reply:
Added a docstring.

For the third column I used `text_to_icon_padding`. This all documented in the ascii art bit.